libs/ui-watt/src/lib/components/modal/watt-modal.component.html

```diff
@@ -15,6 +15,15 @@
 limitations under the License.
 -->
 <ng-template #modal>
+  <button
+    *ngIf="!disableClose"
+    mat-icon-button
+    class="watt-modal-close"
+    type="button"
+    (click)="close(false)"
+  >
+    <watt-icon [label]="closeLabel" name="close"></watt-icon>
+  </button>
```

Comment:
nitpick: we should discuss this with UX (can also be after merge) because the button's look is different than what we have in Watt today. There's no button in Watt that can really be used as a drop-in replacement without making additional adjustments. So we either need to extend Watt to support such a button or use the existing icon button in order to not deviate from Watt's direction.

Reply:
Good point! I talked with UX and it is possible to use the `<watt-button>` here, but it requires changing the size of the icon variant so it matches Figma as well as using the `:focus-visible` selector for focus state. But these are potentially breaking changes, so I will create a different PR for that.